### PR TITLE
fix(docs): Mention alternate valid use of Volid's for containers

### PR DESCRIPTION
**Pull request by lingfish**
Volid's can be used with `template_file_id`.

### Contributor's Note

- [x] I have added / updated documentation in `/docs` for any user-facing features or additions.
- [ ] I have added / updated acceptance tests in `/fwprovider/tests` for any new or updated resources / data sources.
- [ ] I have ran `make example` to verify that the change works as expected.

### Proof of Work
Just a small doco tweak.

### Community Note

- Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

File: docs/resources/virtual_environment_container.md [hunk 1]
```diff
@@ -41,6 +41,8 @@ resource "proxmox_virtual_environment_container" "ubuntu_container" {
 
   operating_system {
     template_file_id = proxmox_virtual_environment_file.latest_ubuntu_22_jammy_lxc_img.id
+    # Or you can use a volume ID, as obtained from a "pvesm list <storage>"
+    # template_file_id = "local:vztmpl/debian-12-standard_12.2-1_amd64.tar.zst"
```
Comment (bpg-dev) [inline docs/resources/virtual_environment_container.md]: to match the example `proxmox_virtual_environment_download_file"` below
```suggestion
    # template_file_id = "local:vztmpl/jammy-server-cloudimg-amd64.tar.gz"
```

File: docs/resources/virtual_environment_container.md [hunk 2]
```diff
@@ -187,6 +189,8 @@ output "ubuntu_container_public_key" {
 - `node_name` - (Required) The name of the node to assign the container to.
 - `operating_system` - (Required) The Operating System configuration.
     - `template_file_id` - (Required) The identifier for an OS template file.
+       This can be a `Volid` from the output of `pvesm list <storage>`, amongst
+       other choices.
```
Comment (bpg-dev) [inline docs/resources/virtual_environment_container.md]:
```suggestion
       The ID format is `<datastore_id>:<content_type>/<file_name>`, for example `local:iso/jammy-server-cloudimg-amd64.tar.gz`. Can be also taken from
       `proxmox_virtual_environment_download_file` resource, or from the output of `pvesm list <storage>`.
```

**Review by bpg — COMMENTED** [review]
Thanks for improving the docs @lingfish!

Would you mind updating your commit message as per the [convention](https://github.com/bpg/terraform-provider-proxmox/blob/main/CONTRIBUTING.md#commit-message-conventions), i.e. "fix(docs): ..."?

**Review by bpg — APPROVED** [review]
Awesome!

LGTM 🚀

**Comment by bpg** [comment]
@all-contributors please add @lingfish for docs

**Pull request merged**